### PR TITLE
Update multi-geo-add-group-with-pdl.md

### PR DESCRIPTION
**Pull request by msbemba**
Added Abbreviation for PDL to avoid confusion.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/4736

File: microsoft-365/enterprise/multi-geo-add-group-with-pdl.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: "Create a Microsoft 365 Group with a specific PDL"
+title: "Create a Microsoft 365 Group with a specific Preferred Data Location (PDL)"
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/multi-geo-add-group-with-pdl.md]:
```suggestion
title: "Create a Microsoft 365 Group with a specific preferred data location (PDL)"
```

File: microsoft-365/enterprise/multi-geo-add-group-with-pdl.md [hunk 2]
```diff
@@ -15,9 +15,9 @@ description: "Learn how to create a Microsoft 365 group with a specified preferr
 ms.custom: seo-marvel-apr2020
 ---
 
-# Create a Microsoft 365 Group with a specific PDL
+# Create a Microsoft 365 Group with a specific Preferred Data Location (PDL)
 
-When users in a multi-geo environment create a Microsoft 365 Group, the group preferred data location is automatically set to that of the user. Global, SharePoint, and Exchange Administrators can create groups in any region they select. 
+When users in a multi-geo environment create a Microsoft 365 Group, the group Preferred Data Location (PDL) is automatically set to that of the user. Global, SharePoint, and Exchange Administrators can create groups in any region they select.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/multi-geo-add-group-with-pdl.md]:
```suggestion
When users in a multi-geo environment create a Microsoft 365 Group, the group preferred data location (PDL) is automatically set to that of the user. Global, SharePoint, and Exchange Administrators can create groups in any region they select. 
```

File: microsoft-365/enterprise/multi-geo-add-group-with-pdl.md [hunk 3]
```diff
@@ -15,9 +15,9 @@ description: "Learn how to create a Microsoft 365 group with a specified preferr
 ms.custom: seo-marvel-apr2020
 ---
 
-# Create a Microsoft 365 Group with a specific PDL
+# Create a Microsoft 365 Group with a specific Preferred Data Location (PDL)
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/multi-geo-add-group-with-pdl.md]:
```suggestion
# Create a Microsoft 365 Group with a specific preferred data location (PDL)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@MikePlumleyMSFT Please merge this PR. Thanks

**Pull request merged**